content/en/blog/2023/0000-00-00-kube-proxy-non-privileged.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: blog
+title: "Kubernetes 1.29: Support for Running kube-proxy in a Non-privileged Container"
+date: 0000-00-00
+slug: kube-proxy-non-privileged
+---
+
+**Author**: Lars Ekman
```

Comment:
For this site, the author information goes in the front matter. See https://github.com/kubernetes/contributor-site/blob/118088ef5115b4665c7b3ce6ee32579dbd494cf7/content/en/blog/2023/contextual-logging-in-kubernetes-1-29.md?plain=1 for an example.